packages/neon/neon/lib/src/theme/theme.dart

```diff
@@ -0,0 +1,77 @@
+import 'package:flutter/material.dart';
+import 'package:meta/meta.dart';
+import 'package:neon/src/theme/neon.dart';
+import 'package:neon/src/utils/hex_color.dart';
+import 'package:nextcloud/nextcloud.dart';
+
+@internal
+late Color themePrimaryColor;
```

Comment:
I really dislike keeping this around. The only place where it is really used is in the PushUtils. I think you already suggested somewhere to just use the Nextcloud blue there, I think this is the right place to do it.